project/src/main/java/com/google/sps/data/PlacesFetcher.java

```diff
@@ -81,18 +82,19 @@
      *
      * @param query A TextSearchRequest with all params to query on
      * @return A PlacesSearchResponse which contains the search results
-     * @throws ApiException
-     * @throws InterruptedException
-     * @throws IOException
+     * @throws FetcherException
      */
     @VisibleForTesting
-    PlacesSearchResult[] getPlacesSearchResults(TextSearchRequest query)
-            throws ApiException, InterruptedException, IOException {
-        return query.await().results;
+    PlacesSearchResult[] getPlacesSearchResults(TextSearchRequest query) throws FetcherException {
+        try {
+            return query.await().results;
+        } catch (ApiException | InterruptedException | IOException e) {
+            throw new FetcherException(e.getMessage());
```

Comment:
Maybe add some more information? 
Something like: "Couldn't fetch from Places API".  
You can then use the exception you threw as the cause to your exception: 
https://docs.oracle.com/javase/7/docs/api/java/lang/Exception.html#Exception(java.lang.String,%20java.lang.Throwable)

Reply:
Good idea